Add ipu-plugin and sriov_cni code (#1)
intel/ipu-opi-plugins: README.md

# IPU OPI Plugins

IPU OPI Plugins are a set of components that expose IPU offload capabilities in a K8s cluster.

> Please follow the [deployment.md](docs/deployment.md) to follow how to deploy and test these components in a supported environment.

Reviewer: ```suggestion
Please follow [deployment.md](docs/deployment.md) to deploy and test these components in a supported environment.
```

> If you would like to contribute to this this project please see the [CONTRIBUTING.md](CONTRIBUTING.md).

Reviewer: ```suggestion
If you would like to contribute to this this project please see the [CONTRIBUTING.md](CONTRIBUTING.md).
```
```suggestion
If you would like to contribute to this project please see the [CONTRIBUTING.md](CONTRIBUTING.md).
```

> If you you would like to contrbute this project please see the [CONTRIBUTING.md](CONTRIBUTING.md).

Reviewer 2: ```suggestion
If you would like to contrbute this project please see the [CONTRIBUTING.md](CONTRIBUTING.md).
```

> If there's any security issues please these [guidelines](SECURITY.md) to report them.

Reviewer 2: ```suggestion
If there's any security issues please see the [guidelines](SECURITY.md) to report them.
```
